pages/Api.ts

```diff
@@ -41,6 +42,8 @@ export class Api {
     internalPage: ApiInternalPage
     otherTestsPage: ApiOtherPage
     agora3002Page: Api3002Page
+    loginPageNegFlow: ApiLoginPageNegFlow
+    static loginPageNegFlow: any
```

Comment:
@YulianH 
1. remove static 
2. type should be equal page name not any

Reply:
done